CHANGELOG.md

```diff
@@ -144,6 +146,32 @@ best long-term interest of our customers.
 as `final`. After the changes in [v1.36.0](#v1360---2022-02), there is no need
 or reason to be extending this class.
 
+**OTHER CHANGES**:
+
+* feat(bigtable): better support for PSC and VPC-SC ([#8458](https://github.com/googleapis/google-cloud-cpp/pull/8458))
+* refactor(bigtable)!: mark InstanceAdminClient as final ([#8449](https://github.com/googleapis/google-cloud-cpp/pull/8449))
+* fix(bigtable): use the appropriate type alias for AsyncRowReader::last_read_row_key_ ([#8437](https://github.com/googleapis/google-cloud-cpp/pull/8437))
+* docs(bigtable): cleanup admin documentation ([#8436](https://github.com/googleapis/google-cloud-cpp/pull/8436))
+* fix(bigtable): ReadRows retries from the `last_scanned_row_key` ([#8423](https://github.com/googleapis/google-cloud-cpp/pull/8423))
+* refactor(bigtable)!: modernize TableAdmin ([#8383](https://github.com/googleapis/google-cloud-cpp/pull/8383))
```

Comment:
and I'd delete this entry. (it is detailed above as a breaking change)

Reply:
Done.